screenpipe-audio/build.rs

```diff
@@ -1,29 +1,91 @@
 #[cfg(target_os = "windows")]
 use std::{env, fs, process::Command};
+use std::{
+    io::Result,
+    process::{Command, Output},
+};
 
 fn main() {
     #[cfg(target_os = "windows")]
     {
-        // Set static CRT for Windows MSVC target
-        if env::var("CARGO_CFG_TARGET_ENV").unwrap_or_default() == "msvc" {
-            println!("cargo:rustc-env=KNF_STATIC_CRT=1");
-            println!("cargo:rustc-flag=-C target-feature=+crt-static");
-        }
+        install_onnxruntime();
+    }
 
-        let url = "https://github.com/microsoft/onnxruntime/releases/download/v1.19.2/onnxruntime-win-x64-gpu-1.19.2.zip";
-        let resp = reqwest::blocking::get(url).expect("request failed");
-        let body = resp.bytes().expect("body invalid");
-        fs::write("./onnxruntime-win-x64-gpu-1.19.2.zip", &body);
-        let status = Command::new("unzip")
-            .args(["onnxruntime-win-x64-gpu-1.19.2.zip"])
-            .status()
-            .expect("failed to execute process");
-        if !status.success() {
-            panic!("failed to install onnx binary");
+    if !is_bun_installed() {
+        install_bun();
+    }
+}
+
+fn is_bun_installed() -> bool {
+    let output = Command::new("bun").arg("--version").output();
+
+    match output {
+        Err(_) => false,
+        Ok(output) => output.status.success(),
+    }
+}
+
+fn run_bun_install_command(command: Result<Output>) {
+    match command {
+        Err(error) => {
+            println!("failed to install bun: {}", error);
+            println!("please install bun manually.");
+        }
+        Ok(output) => {
+            if output.status.success() {
+                println!("bun installed successfully.");
+            } else {
+                println!(
+                    "failed to install bun: {}",
+                    String::from_utf8_lossy(&output.stderr)
+                );
+                println!("please install bun manually.");
+            }
         }
-        fs::rename(
-            "onnxruntime-win-x64-gpu-1.19.2",
-            "../screenpipe-app-tauri/src-tauri/onnxruntime-win-x64-gpu-1.19.2",
+    }
+}
+
+fn install_bun() {
```

Comment:
hmmmmmmm

actually how does this fit with the windows CLI

considering we should do onnx + bun in here actually 

https://github.com/mediar-ai/screenpipe/blob/main/install.ps1

?

what do you think

Reply:
Can you clarify it? 

From my understanding, onnx was already working here, and we needed to add also the bun installing

Inside `install_bun`, we have a command execution installing it with npm. From my understanding, this was happening on pre_build, and we needed to repeat the same thing but on build.rs instead

Reply:
this build.rs assume people are installing the app but would not work for CLI users 

any chance you update the `install.ps1` to install bun too? then will merge

Reply:
fixed. can you take a look @louis030195